app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyNPSScreen.kt

```diff
@@ -0,0 +1,112 @@
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.constraintlayout.compose.ConstraintLayout
+import com.kks.nimblesurveyjetpackcompose.R
+import com.kks.nimblesurveyjetpackcompose.model.SurveyAnswer
+import com.kks.nimblesurveyjetpackcompose.ui.theme.White50
+
+private const val START_INDEX = 0
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun SurveyNPSQuestion(
+    answers: List<SurveyAnswer>,
+    onChooseAnswer: (answers: List<SurveyAnswer>) -> Unit
+) {
+    var selectedIndex by remember { mutableStateOf(answers.indexOfFirst { it.selected }) }
+
+    ConstraintLayout(modifier = Modifier.wrapContentWidth()) {
+        val (nps, notAtAll, extremely) = createRefs()
+        Row(
+            modifier = Modifier
+                .constrainAs(nps) {
+                    start.linkTo(parent.start, 20.dp)
+                    end.linkTo(parent.end, 20.dp)
+                    top.linkTo(parent.top)
+                },
+            horizontalArrangement = Arrangement.Center
+        ) {
+            answers.forEachIndexed { index, surveyAnswer ->
+                Box(contentAlignment = Alignment.Center) {
+                    Surface(
+                        onClick = {
+                            selectedIndex = index
+                            onChooseAnswer(
+                                answers.mapIndexed { index, surveyAnswer ->
+                                    surveyAnswer.copy(selected = index == selectedIndex)
+                                }
+                            )
+                        },
+                        border = BorderStroke(0.5.dp, Color.White),
+                        shape = RoundedCornerShape(
+                            topStart = if (index == START_INDEX) 10.dp else 0.dp,
+                            bottomStart = if (index == START_INDEX) 10.dp else 0.dp,
+                            topEnd = if (index == answers.size - 1) 10.dp else 0.dp,
+                            bottomEnd = if (index == answers.size - 1) 10.dp else 0.dp
+                        ),
+                        color = Color.Transparent,
+                        modifier = Modifier.size(33.dp, 56.dp)
+                    ) { /* Do nothing */ }
+
+                    SurveyBoldText(
+                        text = surveyAnswer.text,
+                        color = if (index <= selectedIndex) {
+                            Color.White
+                        } else {
+                            White50
+                        },
+                        fontSize = 20.sp,
+                        textAlign = TextAlign.Center
+                    )
+                }
+            }
+        }
+        SurveyBoldText(
+            text = stringResource(id = R.string.survey_question_not_at_all_likely),
+            fontSize = 17.sp,
+            modifier = Modifier.constrainAs(notAtAll) {
```

Comment:
Shall we also defined `color` here as well? The [Figma design](https://www.figma.com/file/GjRPOjDyZ6f4EDL3wKarRK/Challenge---Mobile-App?node-id=48%3A79) has the left text faded and the right text vibrant 😄